common/errors.go

```diff
@@ -0,0 +1,39 @@
+package common
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// The canonical errors from the EigenDA gRPC API endpoints.
```

Comment:
Do we need one for timeout?

Reply:
We may add if it's becoming a more clear case to cover